dev/certs/event-gateway-sni-one.crt

```diff
@@ -1,25 +0,0 @@
------BEGIN CERTIFICATE-----
```

Comment:
Are these not needed anymore?

Reply:
i just emptied out the directory and reran the generate commands. maybe they dont generate everything